editor-components/stats/stats7/stats7.tsx

```diff
@@ -107,34 +123,78 @@ class Stats7Page extends BaseStats {
   }
 
   render() {
-    const styling = {
-      backgroundColor: this.getPropValue("backgroundColor"),
-    };
+    const isSubtitleExist = this.castToString(this.getPropValue("subTitle"));
+    const isTitleExist = this.castToString(this.getPropValue("title"));
+    const isDescriptionExist = this.castToString(this.getPropValue("description"));
+    const showDiv = isSubtitleExist || isTitleExist || isDescriptionExist;
+    const itemList = this.castToObject<Item[]>("items")
 
     return (
       <div className={this.decorateCSS("container")} >
-        <div className={this.decorateCSS("max-content")} style={styling}>
-          <div className={this.decorateCSS("stats1-page")}>
-            <div className={this.decorateCSS("title-child")}>
-              <h1 className={this.decorateCSS("title")}>{this.getPropValue("title")}</h1>
-              <h3 className={this.decorateCSS("description")}>{this.getPropValue("description")}</h3>
-            </div>
-            <div className={this.decorateCSS("bottom-child")}>
-              {this.castToObject<ICard[]>("card-content").map(
-                (cardData: any, indexCard: number) => (
-                  <div key={`sts-7-${indexCard}`} className={this.decorateCSS("card")}>
-                    <h5 className={this.decorateCSS("card-data-subtitle")}>{cardData.subtitle}</h5>
-                    <h4 className={this.decorateCSS("card-data-title")}>{cardData.title}</h4>
-                    <p className={this.decorateCSS("card-data-description")}>{cardData.description}</p>
+        <div className={this.decorateCSS("max-content")}>
+
+          {showDiv && <div className={this.decorateCSS("title-child")}>
+            {isSubtitleExist && (
+              <h1 className={this.decorateCSS("subTitle")}>{isSubtitleExist}</h1>
+            )}
+            {isTitleExist && (
+              <h1 className={this.decorateCSS("title")}>{isTitleExist}</h1>
+            )}
+            {isDescriptionExist && (
+              <p className={this.decorateCSS("description")}>
+                {isDescriptionExist}
+              </p>
+            )}
+          </div>}
+
+          {itemList.length > 0 && <div className={this.decorateCSS("progress-container")}>
+            {this.castToObject<Item[]>("items").map((item: Item, index: number) => {
```

Comment:
declare this between redner and return statemens.
```javascript
const items = this.castToObject<Item[]>("items");
```
```suggestion
            {items.map((item: Item, index: number) => {
```